index.html

```diff
@@ -10,5 +10,52 @@
     <div class="page__container">
       <p>The repository is created with the purpose of learning adaptive web design and team collaboration using Git and GitHub.</p>
     </div>
+    <header class="into">
+      <section class="into__top-menu">
+          <picture>
+              <source srcset="images/logotype-pink-white-mobile.png" media="(max-width:320px)">
+              <source srcset="images/logotype-white-tablet.png" media="(max-width:660px)">
+              <img src="images/logotype-white-desktop.png" alt="Glasses and the text is pink under them" class="into__logo">
+          </picture>
+              <nav>
+                  <ul class="into__nav-list">
+                      <li>
+                        <a href="">Головна</a>
+                      </li>
+                      <li>
+                        <a href="">Фотографії</a>
+                      </li>
+                      <li>
+                        <a href="">Конкурс</a>
+                      </li>
+                      <li>
+                        <a href="">HTML Academy</a>
+                      </li>
+                  </ul>
+                      <img src="images/menu-burger.png" alt="menu burger" class="into__menu-burger">
+              </nav>
+          </section>
+
+          <section class="into__content">
+              <h1 class="into__title">Подивись на світ інакше</h1>
+
+              <div class="into__info">
+                <a href="" class="into__button" aria-label="download app">Скачати програму</a>
+                <ul class="into__systems-list">
```

Comment:
The classname is wrong here. The CSS class and file that you've is `blocks/into/__system-list/into__system-list.css` (without 's').